topo_processor/metadata/collection.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+
+import pystac as stac
+
+from .data_type import DataType
+
+GLOBAL_PROVIDERS = [stac.Provider(name="LINZ", description="Land Information New Zealand", roles=["Host"])]
+
+
+class Collection:
+    """
+    collection of items
+    """
+
+    tilte: str
+    description: str
+    license: str
+    data_type: DataType
+    items: List
```

Comment:
`List` needs a type as to what is inside the list.

Reply:
That will create a circular import again where collection.py needs to import item (And we are already importing collection in item.py). Does that mean we need an item factory?